SECP-service/src/main/resources/assets/app/js/profiles/user-profile-controller.js

```diff
@@ -0,0 +1,26 @@
+angular.module('SECP')
+    .controller('UserProfileController', ['$scope', function ($scope) {
+        $scope.IsSaveVisible = false;
+        $scope.IsDisplayNameReadOnly = true;
+        $scope.IsImageFileVisible = false;
+        $scope.IsEditAvailable = true;
+        $scope.EditProfileAvailable = function () {
+            let username = angular.element('#username').html();
```

Comment:
hmm i don't understand the logic here. Please kindly shed some light.

Reply:
For there to be an edit profile button, the profile must be the users. So u need to check first if the profile belongs to the user